### PR TITLE
Ada's changes to diagnostics

### PR DESCRIPTION
**Pull request by rando2**
@aadattoli does this look like everything you expected to see? I think I moved everything around that you had committed last night and this morning!

### CODE REVIEW

**Comment by aadattoli** [comment]
Dear @rando2 , 
Yes everything seems correct! Let me know what you guys think.

Best regards

File: content/10.diagnostics.md [hunk 1]
```diff
@@ -2,11 +2,69 @@
...
+With this test, If CAS12 and its guide find a match in the RNA, then the cut occurs and the patient is positive to the novel Coronavirus. Specifically, when CAS12 is active it also cuts a designed strand containing molecules that change color once they are free [doi.org/10.1101/2020.03.06.20032334. 
+This test can be as fast as 30 minutes and the result can be as simple as a color change on a strip [doi:10.1126/science.abb8400].
+
+Limitation of CRISPR test
+
+Fake-positives due to the specificity of the technique, as the guide RNA can recognise other interspersed sequences on the patient’s genome.
+
+Sierological tests
+
+Despite diagnostic tests based on the detection of the genetic material can be quite sensitive, they cannot provide information about the real extent of the disease. In fact, they would not work on a recovered patient who defeated the virus.
```
[the remark this answers is not in the document]
Comment (aadattoli) [reply content/10.diagnostics.md:54]: with extent I mean: how many people are/have been infected? For instance in Italy infected people (detected with RNA tests) are about 130000. Scientists from the Imperial College (London) estimate that this number can be way higher (in the order of millions).

File: content/10.diagnostics.md [hunk 2]
```diff
@@ -2,11 +2,69 @@
...
+
+Fake-positives due to the specificity of the technique, as the guide RNA can recognise other interspersed sequences on the patient’s genome.
+
+Sierological tests
+
+Despite diagnostic tests based on the detection of the genetic material can be quite sensitive, they cannot provide information about the real extent of the disease. In fact, they would not work on a recovered patient who defeated the virus. 
+In this context, immunity tests can be significantly more informative. 
+In fact, they can help scientists figuring out why the disease has different course among patients, as well as what strategy might work to manage the spread of the infection. 
+Furthermore, sierological tests are crucial for the economic re-start of the hit area. 
+For instance, people that have developed antibodies can return to work prior to the others, helping to develop the herd immunity which will prevent further diffusion of the virus.
```
[the remark this answers is not in the document]
Comment (aadattoli) [reply content/10.diagnostics.md:58]: yes you are correct, just that are not so many paper about it

File: content/10.diagnostics.md [hunk 3]
```diff
@@ -2,11 +2,69 @@
...
+Around the same time the Hong Kong university developed a Quantitative Real Time Reverse Transcription PCR (QRT-tPCR) against two region of the virus, orf1b and N region [doi.org/10.1093/clinchem/hvaa029]. 
+Specifically, this assay was tested on samples coming from two SARS-Cov-2 patients, including a panel of positive and negative controls consisting of RNA extracted from several cultured viruses. 
+The assay uses the N gene to screen patients, while the orf1b is used to confirm the infection [doi.org/10.1093/clinchem/hvaa029].
+In this case the test was designed to detect sabercoviruses of which COVID-19 belongs. 
+Considering that no other sabercoviruses are known to infect human at the moment, if the test comes out positive it means the patient is infected with the virus. 
+However, this test does not allow to discriminate the genetics among different viruses. 
+
+Limitation of PCR tests
+
+The PCR test can be highly sensitive. However, in high-throughput settings it can show several problems:
```
Comment (rando2) [inline content/10.diagnostics.md:28]:
```suggestion
One advantage of PCR tests it their sensitivity; however, several limitations with these tests introduce challenges to high-throughput testing:
```

File: content/10.diagnostics.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rando2) [reply content/10.diagnostics.md:54]: Updated above!

File: content/10.diagnostics.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (rando2) [reply content/10.diagnostics.md:58]: No worries, I didn't mean to make you do math :) Just trying to say I'm assuming we don't know yet what that number is for this disease!

File: content/10.diagnostics.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (aadattoli) [reply content/10.diagnostics.md:54]: couldn't be said better! Thanks!

**Comment by rando2** [comment]
@cbrueffer @grevetjd @kyle-dyson Do any of you have any feedback on this section? (I saw in the "Introduce yourself" thread that you're all interested in diagnostics!)

**Comment by rando2** [comment]
@aadattoli if you don't mind my changes, I can put them into the text -- I think because of my tricks moving your text onto a new branch, you might not be able to add them yourself!

**Comment by aadattoli** [comment]
@eberwine I was wondering whether you would like to have a look at the piece I wrote for diagnostic, before we incorporate it into the text.

**Comment by aadattoli** [comment]
@rando2. ok Sorry, following a webinar. I have to work on the text a little still, as my professor, suggested few changes.

**Comment by rando2** [comment]
> @rando2. ok Sorry, following a webinar. I have to work on the text a little still, as my professor, suggested few changes.

That's great! I was hoping to clean things up to remove some of the pop-out boxes I added by making suggestions, but it doesn't seem to have worked, sadly.

**Comment by aadattoli** [comment]
Dear @rando2, forgive me for having been disappeared. Got lots to do with my own project. I will revise the text today, adding few more things if you agree to that

**Comment by rando2** [comment]
> Dear @rando2, forgive me for having been disappeared. Got lots to do with my own project. I will revise the text today, adding few more things if you agree to that

Hi @aadattoli that sounds fine, just let me know if I can help!

**Comment by aadattoli** [comment]
> > Dear @rando2, forgive me for having been disappeared. Got lots to do with my own project. I will revise the text today, adding few more things if you agree to that
> 
> Hi @aadattoli that sounds fine, just let me know if I can help!

Hey @rando, I complete the changes. I was not able to accept your changes. Not sure why, I will try again now. Thanks for your patience

**Comment by rando2** [comment]
Hi @aadattoli, I think this has to do with the fact that I changed the commit to pretend it was from you (because I reversed your original commit). I will do some investigating and fix it but I'm not sure I'll have time until tomorrow. I'll try to get it merged in ASAP though!

**Comment by aadattoli** [comment]
> Hi @aadattoli, I think this has to do with the fact that I changed the commit to pretend it was from you (because I reversed your original commit). I will do some investigating and fix it but I'm not sure I'll have time until tomorrow. I'll try to get it merged in ASAP though!

Hey @rando2, I rearranged the text here https://github.com/aadattoli/covid19-review/commit/24074772bf6c7e7a1c1bac5a0b63ab87545b39c4. Can you see it?

No worries, whenever you have time. I will try to accept your changes. Best

**Comment by aadattoli** [comment]
@rando2 I am still not able to accept the changes, but I agree to the changes that you proposed. Therefore, if you want to just copy and paste your text in the main MS (when you will feel confident) I am fine with it. Please let me know if you see the other changes I made (aadattoli@2407477).

**Comment by rando2** [comment]
Hi @aadattoli I can see them but I'm not sure how to handle this since we can't merge the suggestions from here in! I think what might make sense is if I just figure out how to commit the suggestions here and then re-arrange per what you did in that commit. Yikes, this one is complicated!

**Comment by agitter** [comment]
Merging in https://github.com/aadattoli/covid19-review/commit/2407477 is going to be messy because it uses 3abc4eb503c4f282249d789712bbead0b501d35d (the third commit from this pull request) as the parent commit instead of the latest commit from the pull request.  I was curious whether it would cleanly, but https://github.com/greenelab/covid19-review/compare/ada...aadattoli:patch-4 shows there will be merge conflicts.

**Comment by aadattoli** [comment]
@rando2 thank you very much for what your doing to fix my text!

**Comment by rando2** [comment]
It is letting me commit now, but not batch commits. Apologies in advance for the ~40 email notifications everyone on this thread is about to get!

**Comment by aadattoli** [comment]
Dear @rando2, 
Thank you so much for all the work u have done to fix the text. Just wanted to tell you that I rearranged stuff differently in here  aadattoli@2407477. Do you want me to create another pull request on the base of the text  you added to the master, so that I can rearrange the text and add few things again?

Best,

Ada

**Pull request merged**